moc_openshift.py

```diff
@@ -314,185 +329,6 @@ def update_user_role_project(self, project_name, user, role, operation):
             mimetype="application/json",
         )
 
-    def generate_quota_name(project_name, quota_def):
```

Comment:
There's a lot of code being removed here (and not re-added). Is this intentional?

Reply:
Unfortunately I included code from the prevues PR.  That includes code that should be in QuotaSupport as well as code the unused class MocOpenShift3x class which were at the root of causing Errors (not warnings/informationals) in pyLint.